src/app/store/middleware/absenderliste-epics.ts

```diff
@@ -41,4 +42,46 @@ export class AbsenderlisteEpics {
         return Observable.of(act);
       });
   }
+
+  savingAbsender = (action: ActionsObservable<any>, store: NgRedux<FormBoxState>) => {
```

Comment:
Dieses Epic ist unnötig. Den ganzen Schritt kann man in changingAbsender machen. CHANGE_ABSENDER muss dann auch nicht asynchron sein.
UPDATE_STORAGE_SELECTED sollte die Id als Parameter bekommen.